src/spikeinterface/qualitymetrics/misc_metrics.py

```diff
@@ -520,18 +520,18 @@ def compute_sliding_rp_violations(
 )
 
 
-def get_synchrony_counts(spikes, synchrony_sizes, all_unit_ids):
```

Comment:
This does change the order in order to have the default. I guess I don't see a way around this, so it's okay.

Reply:
Yeah, I think you're right. The function is only used in `compute_synchrony_metrics`, and this allows people to use `get_synchrony_counts` with the `synchrony_sizes` in another context and keeps the order in tact. Nice.